src/tools/genpolicy/src/main.rs

```diff
@@ -52,5 +53,7 @@ async fn main() {
 
     debug!("Exporting policy to yaml file...");
     policy.export_policy();
+    debug!("Persisting image layers cache...");
```

Comment:
I would move this debug message into try_persist(), just above "self.inner.lock().unwrap()".